<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gameWindow</class>
 <widget class="QWidget" name="gameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_lifes_text_4">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>90</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Player 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_points">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>70</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_points_text">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>70</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Points:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_lifes_num">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>50</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_points_2">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>130</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_lifes_num_2">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>110</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_points_text_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>130</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Points:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_game_level_num">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>160</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_lifes_text_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>110</y>
     <width>26</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Lifes:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_lifes_text">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>50</y>
     <width>26</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Lifes:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_player1">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>30</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Player 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_game_level">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>160</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Game Level:</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>620</width>
     <height>620</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_game_level_2">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>210</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Use WSAD keys to move</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
